index.js

```diff
@@ -23,17 +56,8 @@ module.exports = {
       return "<!-- EMBER_CLI_FASTBOOT_TITLE -->";
     }
 
-    if (type === 'vendor-prefix') {
-      return '// Added from ember-cli-fastboot \n' +
-             'EmberENV.FEATURES = EmberENV.FEATURES || {};\n' +
-             'EmberENV.FEATURES["ember-application-visit"] = true;\n';
-    }
-  },
-
-  included: function() {
```

Comment:
:+1: avoiding `included` hook allows us to have fastboot be available in nested addons (nested addons normally do not have `included` hook called on them).